pages/blog/june-2019-update.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Here are the most significant changes in June.
+---
+
+# June 2019 Update
+
+**Olivier Tassinari**
+
+*July 8, 2019*
+
+Here are the most significant changes in June:
+
+- 💄 We have introduced a new [ButtonGroup](/components/buttons/#grouped-buttons) component.
+  ![ButtonGroup](/static/blog/june-2019-update/button-group.png)
+
+- 💄 We have introduced a new [Slider](/components/slider/) component.
+  ![Slider](/static/blog/june-2019-update/slider.png)
+
+- 💄 We have introduced a new [TextareaAutosize](/components/textarea-autosize/) component.
+  <video src="/static/blog/june-2019-update/textarea-autosize.mp4" controls loop autoplay></video>
+
+But this summary is just scratching the surface. We have accepted 198 commits from 71 different contributors. We have changed 5,384 files with 26,199 additions and 18,097 deletions.
+
+## Our roadmap intent for July
+
+*(We'll do our best, no guarantee!)*
+
+- We will keep working on a new Tree View component.
```

Comment:
are these new components part of Google/Angular Material Design or are they extra/extension components unique to material-ui? I think I remember seeing in the docs that it's indicated which ones are core and which ones are extra?

Reply:
Tree View falls under 
> extra/extension components unique to material-ui

Reply:
Also, the Material Design team seems to be interested in (from one discussion I had with them) writing guidelines for advanced (but already established) components like the tree view.

Reply:
That isn't too surprising, most UI frameworks will end up with certain components/widgets in their core because it makes them far more useful and the competition is *fierce*. The examples I always think of are [GTK](https://developer.gnome.org/gtk3/stable/gtkobjects.html), [Qt](https://doc.qt.io/qt-5/widget-classes.html), and [ExtJS](https://en.wikipedia.org/wiki/Ext_JS#GUI_controls_(components)). Seems like it would be possible just to pick one of their component lists and follow that as a roadmap, haha.